sensing/radar_static_pointcloud_filter/src/radar_static_pointcloud_filter_node/radar_static_pointcloud_filter_node.cpp

```diff
@@ -0,0 +1,172 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "radar_static_pointcloud_filter/radar_static_pointcloud_filter_node.hpp"
+
+#include <geometry_msgs/msg/transform_stamped.hpp>
+#include <geometry_msgs/msg/twist_with_covariance.hpp>
+#include <geometry_msgs/msg/vector3_stamped.hpp>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace
+{
+template <class T>
+bool update_param(
+  const std::vector<rclcpp::Parameter> & params, const std::string & name, T & value)
+{
+  const auto itr = std::find_if(
+    params.cbegin(), params.cend(),
+    [&name](const rclcpp::Parameter & p) { return p.get_name() == name; });
+  // Not found
+  if (itr == params.cend()) {
+    return false;
+  }
+  value = itr->template get_value<T>();
+  return true;
+}
+
+geometry_msgs::msg::Vector3 getVelocity(const radar_msgs::msg::RadarReturn & radar)
+{
+  return geometry_msgs::build<geometry_msgs::msg::Vector3>()
+    .x(radar.doppler_velocity)
+    .y(0.0)
+    .z(0.0);
```

Comment:
Let me check :pray: 
Can radar get the velocity in polar coordinates?
If so, it maybe better to divide x,y vector.

Reply:
fixed at [0f178cc](https://github.com/autowarefoundation/autoware.universe/pull/2362/commits/0f178cc03a40c3fb271b51333d91b601affe58c0)

Reply:
@scepter914 
> Can radar get the velocity in polar coordinates?
Please answer this question. :pray:

Reply:
The velocity is in polar coordinates, so fixed to divide x and y vector.

Reply:
Thank you
> fixed at [0f178cc](https://github.com/autowarefoundation/autoware.universe/pull/2362/commits/0f178cc03a40c3fb271b51333d91b601affe58c0)

Is this mistaked since this commit id is unknown?
I think correct id is https://github.com/autowarefoundation/autoware.universe/pull/2362/commits/86fd1998bf44262c813c01c33f261e4bc2246549